core/vtk/ttkAlgorithm/ttkAlgorithm.cpp

```diff
@@ -165,18 +150,20 @@ ttk::Triangulation *ttkAlgorithm::InitTriangulation(void *key,
     }
 
     // Cells
-    if(cells->GetNumberOfCells() > 0) {
-#ifdef TTK_CELL_ARRAY_NEW
```

Comment:
I think this `ifdef` should not be removed for the elder cell array api to be supported (PV 5.7)
There is two `setInputCell` methods and the one used depends on the version of ParaView / VTK we are using.
Elder version represent cells with a single flat array. You may add the validation code only in the newer layout using both `connectivity` and `offset` but you need to still use the elder one when `TTK_CELL_ARRAY_NEW` is not define for ParaView 5.7.

Reply:
Hi, I can put these guards back in, but is it worth keeping that old code around? We are moving to the next version anyway. Also, as Peter found out, building TTK only with a VTK repo (no paraview) doesn't work right now anyway.

Reply:
you need to be on a VTK commit close to the one used by ParaView 5.7 to use an "old" VTK.
Whether we want to still keep the compatibility with PV 5.7 and corresponding VTK is not up to me.